app/themes/index.ts

```diff
@@ -15,6 +15,21 @@ declare module "@mui/material" {
     extends FederalTypographyVariantsOptions {}
 }
 
+declare module "@mui/material/Alert" {
+  interface AlertPropsColorOverrides {
+    error: false;
+    warning: false;
+    info: false;
+    success: false;
+
+    red: true;
+    orange: true;
+    yellow: true;
+    blue: true;
+    green: true;
```

Comment:
Why do we have both yellow and orange alerts ?

Reply:
I think I initially made a mistake and thought we'd actually have two, but I see it's already corrected in later commits 👍